.github/workflows/continuous-deploy.yml

```diff
@@ -620,30 +715,97 @@ jobs:
+
+          # Copy the file from the pod to your local system
+          echo "Copying file from pod..."
+          mkdir -p $LOCAL_PATH
+          kubectl -n $NAMESPACE cp $POD_NAME:$MOUNT_PATH/$OUTPUT_FILE $LOCAL_PATH/$OUTPUT_FILE
+          kubectl -n $NAMESPACE cp $POD_NAME:$MOUNT_PATH/$COVERAGE_FILE $LOCAL_PATH/$COVERAGE_FILE
+
+          # Clean up: delete the temporary pod
+          echo "Cleaning up..."
+          kubectl -n $NAMESPACE delete pod $POD_NAME
+
+          echo "Done!"
+        env:
+          PVC_NAME: cloudapi-pytest-regression
+          POD_NAME: pytest-results-regression
+          CONTAINER_IMAGE: busybox
+          MOUNT_PATH: /mnt
+          LOCAL_PATH: ./pytest-regression
+          NAMESPACE: dev-cloudapi
+
+      - name: Publish Pytest Report
         uses: mikepenz/action-junit-report@v4
-        if: ${{ github.event_name != 'workflow_dispatch' || (github.event_name == 'workflow_dispatch' && github.event.inputs.run-tests != 'false') }}
+        if: steps.pytest.outcome == 'success'
         with:
-          report_paths: "./test_output.xml"
+          check_name: JUnit Test Report
+          report_paths: "./pytest/test_output.xml"
           fail_on_failure: true
           detailed_summary: true
           require_passed_tests: true
 
+      - name: Publish Pytest Regression Report
+        uses: mikepenz/action-junit-report@v4
+        if: steps.pytest-init-regression.outcome == 'success' || steps.pytest-run-regression.outcome == 'success'
+        with:
+          check_name: JUnit Test Report Regression
+          report_paths: "./pytest-regression/test_output.xml"
+          fail_on_failure: true
+          detailed_summary: true
+          require_passed_tests: true
+
+      - name: Pytest coverage comment
+        if: steps.pytest.outcome == 'success'
+        uses: MishaKav/pytest-coverage-comment@v1.1.51
+        with:
+          pytest-coverage-path: ./test_coverage.txt
+          junitxml-path: ./pytest/test_output.xml
+          create-new-comment: true
+          title: "K8s Test Coverage"
+          # Resolves `Warning: Your comment is too long (maximum is 65536 characters), coverage report will not be added.`
+          # hide-report: ${{ github.event_name != 'pull_request' }}
+
       - name: Pytest coverage comment
-        if: (
-          github.event_name == 'pull_request' ||
-          (github.event_name == 'workflow_dispatch' && github.event.inputs.run-tests != 'false')
-          )
+        if: steps.pytest-init-regression.outcome == 'success' || steps.pytest-run-regression.outcome == 'success'
         uses: MishaKav/pytest-coverage-comment@v1.1.51
         with:
           pytest-coverage-path: ./test_coverage.txt
-          junitxml-path: ./test_output.xml
+          junitxml-path: ./pytest-regression/test_output.xml
           create-new-comment: true
+          title: "K8s Regression Test Coverage"
```

Comment:
I don't think the changes to the coverage comment have worked. If you look at the comment (https://github.com/didx-xyz/aries-cloudapi-python/pull/806#issuecomment-2146975836) it no longer contains a coverage report or % test coverage.

We in any case don't need a coverage report for regression tests, and it should instead be based on the CICD / Test results, instead of Deploy to EKS. I guess that can be changed in another PR, so will approve

Reply:
Should have fixed it now in anyway. Forgot to update the `pytest-coverage-path`.